### DISCUSSION TITLE
How do we change the cursor, followpoint, or hitsound?

### DISCUSSION THREAD

**Discussion by Megumiiiiii** [ppy/osu#24768] [Q&A] [answered]
In the stable version, we just need to change it in the /osu!/skin/ folder, where the folder is located for lazer?

**Comment by cdwcgt** [comment]
open osk by lazer or drag it to lazer's windows then skin will import

**Reply by Megumiiiiii** [reply]
so there is no option to edit directly in the lazer folder?

**Reply by cdwcgt** [reply, marked as answer]
lazer use realm database to store file for hashed storage. 

At present, the file system of lazer cannot be edited directly.

Also for beatmapset

**Reply by Megumiiiiii** [reply]
thank you

**Comment by Megumiiiiii** [comment]
![Screenshot_82](https://github.com/ppy/osu/assets/98658943/e1e279ad-6d52-4be5-8376-e558869a5809)
also how to disable this in lazer?

**Reply by sineplusx** [reply]
There is no option in the lazer settings for that. A possible workaround is to disable it for every skin in the `skin.ini` config:

In the `[General]` section set `AllowSliderBallTint: 0`

**Reply by Megumiiiiii** [reply]
thank you
